src/components/Badge/README.md

```diff
@@ -182,12 +183,16 @@ Keep in mind that seeing this badge can feel stressful for merchants so it shoul
 
 ### Critical badge
 
-<!-- example-for: android, ios -->
+<!-- /content-for -->
```

Comment:
Why this change?

Reply:
it wasn't showing up in story book without `/content-for` i think that's defined explicitly in the storybook setup

Reply:
Looked through `polaris-readme-loader` is there a better way to surface this within storybook?

Reply:
Delete this line.

Note that the other examples that have contents for the web tab don't wrap it in anything.

Reply:
makes sense, removing the comment makes it available for all examples, thank you!